gatsby/content/projects/bridges/linkedin-matrix.mdx

```diff
@@ -0,0 +1,29 @@
+---
+layout: project
+title: linkedin-matrix
+categories: 
+ - bridge
+description: A LinkedIn Messaging <-> Matrix bridge
+author: sumner
+maturity: Beta
+language: Python
+license: Apache 2.0
+repo: https://gitlab.com/beeper/linkedin
+room: "#linkedin-matrix:nevarro.space"
+featured: true
+bridges: LinkedIn Messaging
```

Comment:
Only sticky point to me was looking into whether this should be `LinkedIn` vs `LinkedIn Messaging`.

The current `LinkedIn Messaging` seems on target to what they call it and doesn't bridge other parts of LinkedIn so I think so this is good 👍